site/hugo.toml

```diff
@@ -0,0 +1,173 @@
+github_project_repo = ""
+
+# Specify a value here if your content directory is not in your repo's root directory
+github_subdir = "site"
+
+# Uncomment this if your GitHub repo does not have "main" as the default branch,
+# or specify a new value if you want to reference another branch in your GitHub links
+github_branch= "main"
+
+# Google Custom Search Engine ID. Remove or comment out to disable search.
+gcs_engine_id = "d72aa9b2712488cc3"
+
+# Enable Lunr.js offline search
+offlineSearch = false
+
+# Enable syntax highlighting and copy buttons on code blocks with Prism
+prism_syntax_highlighting = false
+
+[[params.versions]]
+version = "v0.1"
+url = "https://etcd.aenix.io/docs/v0.1"
+
+[params.copyright]
+  authors = "Aenix.io | "
+  from_year = 2024
+
+# User interface configuration
+[params.ui]
+#  Set to true to disable breadcrumb navigation.
+breadcrumb_disable = false
+# Set to false if you don't want to display a logo (/assets/icons/logo.svg) in the top navbar
+navbar_logo = true
+# Set to true if you don't want the top navbar to be translucent when over a `block/cover`, like on the homepage.
+navbar_translucent_over_cover_disable = false
+# Enable to show the side bar menu in its compact state.
+sidebar_menu_compact = false
+# Set to true to hide the sidebar search box (the top nav search box will still be displayed if search is enabled)
+sidebar_search_disable = false
+
+# Adds a H2 section titled "Feedback" to the bottom of each doc. The responses are sent to Google Analytics as events.
+# This feature depends on [services.googleAnalytics] and will be disabled if "services.googleAnalytics.id" is not set.
+# If you want this feature, but occasionally need to remove the "Feedback" section from a single page,
+# add "hide_feedback: true" to the page's front matter.
+[params.ui.feedback]
+enable = true
+# The responses that the user sees after clicking "yes" (the page was helpful) or "no" (the page was not helpful).
+yes = 'Glad to hear it! Please <a href="https://github.com/aenix-io/etcd-operator/issues/new">tell us how we can improve</a>.'
+no = 'Sorry to hear that. Please <a href="https://github.com/aenix-io/etcd-operator/issues/new">tell us how we can improve</a>.'
+
+# Adds a reading time to the top of each doc.
+# If you want this feature, but occasionally need to remove the Reading time from a single page,
+# add "hide_readingtime: true" to the page's front matter
+[params.ui.readingtime]
+enable = true
+
+[params.links]
+# End user relevant links. These will show up on left side of footer and in the community page if you have one.
+[[params.links.user]]
+name = "GitHub Discussions"
+url = "https://github.com/aenix-io/etcd-operator/discussions"
+icon = "fab fa-github"
+desc = "Discussions and help from your fellow users!"
+[[params.links.user]]
+name = "Meeting Recordings"
+url = "TODO"
+icon = "fa fa-video"
+desc = "Watch recordings of our community meetings."
```

Comment:
Any ideas where to publish meeting recordings?

eg. KubeVirt projct has specific telegram channel, we could do the same

Reply:
Youtube?
Telegram is fine.

Reply:
Internally discussed that kubevirt has telegram channel with links to youtube. And youtube has a playlist. So it would be better to upload meeting records to youtube

Reply:
Okay I've created youtube channel and playlist

https://www.youtube.com/@etcd-operator
https://www.youtube.com/playlist?list=PLJtQvuQ_bJl1dbKiI8zp6KXn9b4NEyQZh

Reply:
```suggestion
name = "Meeting Recordings"
url = "https://www.youtube.com/playlist?list=PLJtQvuQ_bJl1dbKiI8zp6KXn9b4NEyQZh"
icon = "fa fa-video"
desc = "Watch recordings of our community meetings."
```